src/main/cpp/hw/DragonTalonFX.cpp

```diff
@@ -213,68 +203,10 @@ void DragonTalonFX::UpdateFramePeriods
-{
-	return;
-	/**
-	switch ( priority )
-	{
-		case HIGH:
-			UpdateFramePeriods( StatusFrameEnhanced::Status_1_General, 10 );
-			UpdateFramePeriods( StatusFrameEnhanced::Status_2_Feedback0, 20 );
-			UpdateFramePeriods( StatusFrameEnhanced::Status_3_Quadrature, 100 );
-			UpdateFramePeriods( StatusFrameEnhanced::Status_4_AinTempVbat, 150 );
-			UpdateFramePeriods( StatusFrameEnhanced::Status_8_PulseWidth, 120 );
-			UpdateFramePeriods( StatusFrameEnhanced::Status_10_Targets, 120 );
-			UpdateFramePeriods( StatusFrameEnhanced::Status_11_UartGadgeteer, 120 );
-			UpdateFramePeriods( StatusFrameEnhanced::Status_12_Feedback1, 120 );
-			UpdateFramePeriods( StatusFrameEnhanced::Status_13_Base_PIDF0, 120 );
-			UpdateFramePeriods( StatusFrameEnhanced::Status_14_Turn_PIDF1, 120 );
-			UpdateFramePeriods( StatusFrameEnhanced::Status_15_FirmareApiStatus, 200 );
-			UpdateFramePeriods( StatusFrameEnhanced::Status_Brushless_Current, 200 );
-			break;
-
-		case MEDIUM:
-			UpdateFramePeriods( StatusFrameEnhanced::Status_1_General, 60 );
-			UpdateFramePeriods( StatusFrameEnhanced::Status_2_Feedback0, 120 );
-			UpdateFramePeriods( StatusFrameEnhanced::Status_3_Quadrature, 150 );
-			UpdateFramePeriods( StatusFrameEnhanced::Status_4_AinTempVbat, 200 );
-			UpdateFramePeriods( StatusFrameEnhanced::Status_8_PulseWidth, 150 );
-			UpdateFramePeriods( StatusFrameEnhanced::Status_10_Targets, 150 );
-			UpdateFramePeriods( StatusFrameEnhanced::Status_11_UartGadgeteer, 150 );
-			UpdateFramePeriods( StatusFrameEnhanced::Status_12_Feedback1, 150 );
-			UpdateFramePeriods( StatusFrameEnhanced::Status_13_Base_PIDF0, 150 );
-			UpdateFramePeriods( StatusFrameEnhanced::Status_14_Turn_PIDF1, 150 );
-			UpdateFramePeriods( StatusFrameEnhanced::Status_15_FirmareApiStatus, 200 );
-			UpdateFramePeriods( StatusFrameEnhanced::Status_Brushless_Current, 200 );
-			break;
-
-		case LOW:
-			UpdateFramePeriods( StatusFrameEnhanced::Status_1_General, 120 );
-			UpdateFramePeriods( StatusFrameEnhanced::Status_2_Feedback0, 200 );
-			UpdateFramePeriods( StatusFrameEnhanced::Status_3_Quadrature, 200 );
-			UpdateFramePeriods( StatusFrameEnhanced::Status_4_AinTempVbat, 200 );
-			UpdateFramePeriods( StatusFrameEnhanced::Status_8_PulseWidth, 200 );
-			UpdateFramePeriods( StatusFrameEnhanced::Status_10_Targets, 200 );
-			UpdateFramePeriods( StatusFrameEnhanced::Status_11_UartGadgeteer, 200 );
-			UpdateFramePeriods( StatusFrameEnhanced::Status_12_Feedback1, 200 );
-			UpdateFramePeriods( StatusFrameEnhanced::Status_13_Base_PIDF0, 200 );
-			UpdateFramePeriods( StatusFrameEnhanced::Status_14_Turn_PIDF1, 200 );
-			UpdateFramePeriods( StatusFrameEnhanced::Status_15_FirmareApiStatus, 200 );
-			UpdateFramePeriods( StatusFrameEnhanced::Status_Brushless_Current, 200 );
-			break;
-
-		default:
-		break;
-
-	}
-	**/
-}
 
 void DragonTalonFX::Set(double value)
 {
-	DutyCycleOut out{value};
-	Set(out);
+	m_controller[0]->Set(value);
```

Comment:
Should this be hardcoded to zero or be able to use the different slots in the controller?  Default slot is 0, but can be specified to be a different slot?

Reply:
There are 2 set methods.  This is the one that assumes the zero slot (helps keep it in sync with the REV stuff) and is the one we normally use (we don't normally use the other slots).